src/SWP/Bundle/MultiTenancyBundle/EventListener/TenantSubscriber.php

```diff
@@ -75,8 +76,10 @@ protected function addTenant(LifecycleEventArgs $args)
                 return;
             }
 
-            if (null === $this->tenantContext) {
-                $this->tenantContext = $this->container->get('swp_multi_tenancy.tenant_context');
+            if ($this->phpcrEnabled) {
```

Comment:
I wonder why it's necessary to set the code of the tenant if phpcr is enabled. Looks weird to call a setter with different types of argument.